pkg/setting/setting.go

```diff
@@ -451,6 +451,8 @@ type Cfg struct {
 	// Access Control
 	RBACEnabled         bool
 	RBACPermissionCache bool
+	// Enable Permission scope validation during role creation and provisioning
+	ScopeValidationEnabled bool
```

Comment:
Nit. to be consistent :)
```suggestion
	RBACScopeValidationEnabled bool
```

Reply:
As discussed maybe better use permissions instead.